File: 268-MissingNumber/268-MissingNumber.py
# Last updated: 10/22/2025, 12:09:39 AM
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        nums.sort()
        for i in range(len(nums)):
            if nums[i] != i:
                return i
        return len(nums)
               
        

        

File: 268-MissingNumber/268-MissingNumber.js
// Last updated: 10/26/2025, 3:53:04 PM
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let sum = 0
    let expected = (nums.length*(nums.length+1))/2
    for(const num of nums){
        sum+=num
    }
    return expected - sum
    
};